airbyte-cron/src/main/java/io/airbyte/cron/selfhealing/DefinitionsUpdater.java

```diff
@@ -69,7 +69,7 @@ void updateDefinitions() {
           remoteDefinitionsProvider.getDestinationDefinitions().size());
 
       try {
-        final ApplyDefinitionsHelper applyHelper = new ApplyDefinitionsHelper(configRepository, remoteDefinitionsProvider);
+        final ApplyDefinitionsHelper applyHelper = new ApplyDefinitionsHelper(configDatabase, configRepository, remoteDefinitionsProvider);
```

Comment:
Note that this is also used in BootloaderApp, and cloud's equivalent bootloader as well